pkg/apis/v1beta1/nodeclaim_conversion.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"context"
+
+	"knative.dev/pkg/apis"
+)
+
+func (in *NodeClaim) ConvertTo(_ context.Context, _ apis.Convertible) error { return nil }
```

Comment:
We don't need to implement this on the v1beta1 API?

Reply:
No, since the conversion is happening from v1. We only need to implement the hub version. https://github.com/knative/pkg/blob/3f6a546ac3a42c8237ea0a8b00f5f1fd80260aa8/webhook/resourcesemantics/conversion/controller.go#L64